lib/sprockets/sass/v3/importer.rb

```diff
@@ -124,7 +124,7 @@ def call_processor_input(processor, context, input, processors)
         def handle_complex_process_result(context, result, processors)
           data = result[:data] if result.key?(:data)
           context.metadata.merge!(result)
-          context.metadata.delete(:data)
+          # context.metadata.delete(:data)
```

Comment:
Since it's urgent we should separate this into another PR and I'll merge immediately while we discuss the Less processing